alpha/lib/model/entry.php

```diff
@@ -2794,7 +2794,40 @@ public function postUpdate(PropelPDO $con = null)
 				$redirectEntry->save();
 			}
 		}
-		
+
+		//Has entitledUserPuserEdit/Publish changed for current entry
+		if (  isset($this->oldCustomDataValues['']) &&
+			($this->oldCustomDataValues['']['entitledUserPuserEdit'] != $this->getEntitledPusersEdit() ||
```

Comment:
Add function getOldCustomDataValues(fieldName,namespace='')
This function should return value or null if not exist